src/adapter/src/coord/sequencer/inner.rs

```diff
@@ -5084,6 +5083,12 @@ impl Coordinator {
         match cached_stats {
             Ok(stats) => Ok(Box::new(stats)),
             Err(mz_ore::future::TimeoutError::DeadlineElapsed) => {
+                warn!(
```

Comment:
How bad is it when this fails? Does it indicate storage isn't working as intended? If this is a thing we expect to see in normal operation due to expected storage performance variance, it may be better as an `info!`.

Reply:
That's a good question! To me it's certainly surprising to have it fail---these stats should be available and it should be cheap to get them---but maybe it's not surprising at the level of `warn!`. Happy to switch it to `info!`.

Reply:
So I ended up taking this logging out entirely, because the [`.with_subscriber(&optimizer_trace)` call](https://github.com/MaterializeInc/materialize/blob/0b596d55a6c8f493b63e93c11f7cc719aad61997/src/adapter/src/coord/sequencer/inner.rs#L2988) ends up eating all of these logs anyway. A fix would mean changing how `OptimizerTrace` works, which feels like too much to think about now. (Maybe something to address in the optimizer refactor?)